localstack/services/ssm/provider.py

```diff
@@ -0,0 +1,171 @@
+import json
+import time
+from abc import ABC
+from typing import Optional
+
+from localstack.aws.api import RequestContext
+from localstack.aws.api.ssm import (
+    Boolean,
+    DeleteParameterResult,
+    GetParameterResult,
+    GetParametersResult,
+    LabelParameterVersionResult,
+    ParameterLabelList,
+    ParameterName,
+    ParameterNameList,
+    PSParameterName,
+    PSParameterVersion,
+    PutParameterRequest,
+    PutParameterResult,
+    SsmApi,
+)
+from localstack.services.moto import call_moto, call_moto_with_request
+from localstack.utils.aws import aws_stack
+from localstack.utils.bootstrap import is_api_enabled
+
+
+class SsmProvider(SsmApi, ABC):
+    @staticmethod
+    def _has_secrets(names: ParameterNameList) -> Boolean:
+        maybe_secret = next(
+            filter(lambda n: n.startswith("/aws/reference/secretsmanager"), names), None
```

Comment:
nit: string `"/aws/reference/secretsmanager"` is used twice - could be put into a constant like `PARAM_PREFIX_SECRETSMANAGER` at the global scope in this file.